File: 2P/EST/Provas/PROVA2/No.cs
public class No
{
    private char valor;
    private No? proximo = null;

    public char Valor { get=> valor; set => valor = value; }
    public No? Proximo { get => proximo; set => proximo = value; }

    public No(char valor)
    {
        this.valor = valor;
    }
}

File: 4P/IA/arvores/No.js
class No {
    valor = "";
    filhos = [];

    constructor(valor) {
        this.valor = valor;
    }

    adicionarFilho(no) {
        if (no instanceof No) {
            this.filhos.push(no);
        } else {
            throw new Error("O filho deve ser um nó.");
        }
    }
}

export default No;